codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -151,38 +151,39 @@ update these file paths accordingly.
 
   /// The entrance point of examining and code signing an engine artifact.
   Future<void> validateAll() async {
+    for (String passwordFilePath in [
+      codesignAppstoreIDFilePath,
+      codesignTeamIDFilePath,
+      appSpecificPasswordFilePath,
+    ]) {
+      await readPassword(passwordFilePath);
+    }
+    if (availablePasswords.containsValue('')) {
+      throw CodesignException('certian passwords are missing. \n'
+          'make sure you have provided <CODESIGN_APPSTORE_ID>, <CODESIGN_TEAM_ID>, and <APP_SPECIFIC_PASSWORD>');
+    }
```

Comment:
maybe something like:

`codesignAppstoreId = readPassword(codesignAppstoreIDFilePath);`

 and remove  all the other parts.

Reply:
Sure, thanks for the insights! Todo for myself is to update encrypted password files and update recipe.